tests/default.nix

```diff
@@ -31,8 +32,15 @@ let
   callTests = lib.callPackagesWith autoArgs;
 
   selfPackages = self.packages.${system};
+
+  # For tests that CI should only build on one system,
+  # This is true when on that system.
+  #
+  # TODO: consider refactoring tests/default.nix so that some tests are
+  # defined by it, while others are defined elsewhere...
+  buildForThisSystem = allSystems || system == "x86_64-linux";
```

Comment:
Might be nice to make the fallback checking against a list if we want to set it up to support a `whitelist` of systems we always run on. Could help if the CI situation improves for `aarch64-darwin` down the road.

Reply:
Ah, if these are just platform agnostic then that's fine.